test/helpers/constants.js

```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'https://localhost:8080';
```

Comment:
This needs to be configurable, from an environment variable, with a sane default.